.github/container/install-efa.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+function copy_libefa_from() {
+    PACKAGE_NAME=$1
+    EXTRACT_FOLDER="${PACKAGE_NAME}_"
+    # unpack downloaded package and copy libefa* stuff to /usr/lib    
+    dpkg -x ${PACKAGE_NAME}* ${EXTRACT_FOLDER}
+    # copy all libefa stuff to /usr/lib
+    for n in $(find ${EXTRACT_FOLDER}/ -name libefa*)
+    do
+        mkdir -p "$(dirname $(echo ${n} | cut -d'_' -f2-))"
+        cp -a "${n}" $(echo ${n} | cut -d'_' -f2-)
+    done
+}
+
+set -ex
+
+# Update distro
+apt-get update
+
+# Workaround to have libefa* for Amazon EFA: 
+# for some reason NVIDIA/Mellanox's OFED distribution does not contain libefa*
+# Temporary solution:
+# 1. download libibverbs-dev and ibverbs-providers that contains required libefa*
+# 2. unpack them
+# 3. copy libefa to /usr/lib
+
+WORKDIR=$(mktemp -d)
+pushd ${WORKDIR}
+
+# just in case remove related packages that contains libefa
+apt-get remove -y libibverbs-dev ibverbs-providers
```

Comment:
This would make the execution outcome of install-efa/ofed.sh order-dependent, and this order is not explicit.
Why do we have to remove the two packages if they are already installed?

Reply:
Could we at least raise a warning if the two packages are found pre-existing in the system? Or maybe add an argument `--remove-existing-ibverbs` using shell script options?